client/blocks/app-banner/index.jsx

```diff
@@ -134,6 +135,8 @@ export class AppBanner extends Component {
 			switch ( currentSection ) {
 				case GUTENBERG:
 					return `intent://post/#Intent;scheme=${ scheme };package=${ packageName };end`;
+				case HOME:
```

Comment:
For this to work, I believe we'll need to add `home` [here in the Android app's manifest](https://github.com/wordpress-mobile/WordPress-Android/blob/55bb3298d72aadc5850bc0ee2e5ce03a2f4136fe/WordPress/src/jetpack/AndroidManifest.xml#L24-L38). A deep link to the home section isn't currently in the WordPress app either (judging from [this list](https://github.com/wordpress-mobile/WordPress-Android/blob/a96b4ac411f55e287665cc643bcbb31f970ed4eb/WordPress/src/main/AndroidManifest.xml#L80-L91)), so we'll likely also need to figure out a way to link the intent to the app's home page. This isn't a blocker to this PR, but I wanted to note it as something we'll need to work on down the road.

Reply:
Aha. Thank you for sharing this. This makes sense. I will add this to my todo list to manage. 🙇🏻 

In theory, I suppose only _need_ to add this for the Jetpack app, unless we just want the ability for WordPress app. There is not a banner linking to WordPress app's home to my knowledge.

Reply:
For posterity, https://github.com/wordpress-mobile/WordPress-Android/pull/17239 adds this `home` intent.